packages/InteractiveProfilingTool-Core.package/IPTReport.class/instance/initializeWithUserCallsRootsNode..st

```diff
@@ -0,0 +1,6 @@
+initialization
+initializeWithUserCallsRootsNode: aNode
+
+	aNode
```

Comment:
Vielleicht können wir in `findRootForUserCalls` direkt eine leere Collection anstelle von `nil` zurückgeben? Ist ja auch im Erfolgsfall eine Collection, da scheint mir das ein sinnvoller Default zu sein.

Reply:
Bzw. warum gibt `findRootForUserCalls` eigentlich nur einen Knoten zurück. Ich dachte, das können mehrere sein – deshalb ist `userCallsRoots` ja eigentlich auch eine Collection.

Reply:
Ich stimme zu. Das naming sollte widerspiegeln, dass hier nur eine root node gesetzt wird. Wieso ist das überhaupt eine collection?

Reply:
Bitte wirf einen Blick auf die Kommentare oben und auf #52. Ja, es ergibt keinen Sinn, dass das eine Collection ist. Aber es war schon immer eine Collection, deshalb haben wir uns entschieden, das an dieser Stelle nicht zu behandeln.